wrangler-docs/functions/json.md

```diff
@@ -0,0 +1,207 @@
+the json object or json array.
+ - `json-object` and `json-array` represent different collections.
+ - `json-null` represents a null element of json.
+
+## Example data
+
+Upload to the workspace `json` an input record such as:
+```
+{
+    "name": {
+        "fname": "Joltie",
+        "lname": "Root",
+        "mname": null
+    },
+    "coordinates": [
+        12.56,
+        45.789
+    ],
+    "numbers": [
+        1,
+        2.1,
+        3,
+        null,
+        4,
+        5,
+        6,
+        null
+    ],
+    "moves": [
+        { "a": 1, "b": "X", "c": 2.8},
+        { "a": 2, "b": "Y", "c": 232342.8},
+        { "a": 3, "b": "Z", "c": null},
+        { "a": 4, "b": "U"}
+    ],
+    "integer": 1,
+    "double": 2.8,
+    "float": 45.6,
+    "aliases": [
+        "root",
+        "joltie",
+        "bunny",
+        null
+    ]
+}
+```
+
+Below is an example malformed json described as `malformed_json` that is missing a comma.
+
+```
+    {
+      a : 1,
+      b : 2.0,
+      c : test
+      d : true
+    }
+```
+
+## Parse
+Parses a json into `json-element`. If there are any issues in parsing, the function would return
+`json-null`
+
+### Namespace
+`json`
+
+### Input
+`json-string`
+
+### Output
+`json-element`
+
+### Example
+
+if `json` has the input json specified, then the result of the operation would return `json-element`
+that contains a parsed json.
+
+```
+  set-column parsedjson json:Parse(json)
+```
+
+When a malformed json `malformed_json` is parsed, then the result of operation is `json-null`
+
+```
+  set-column malformedjson json:Parse(malformed_json)
+```
+
+## IsValid
+Returns `true` if json is valid, else `false`
+
+### Namespace
+`json`
+
+### Input
+`json-string`
+
+### Output
+boolean(`true`|`false`)
+
+### Example
+
+if `json` has the input json specified, then the result of the operation would return `true`
+
+```
+  set-column validjson json:IsValid(json)
+```
+
+When a malformed json `malformed_json` is parsed, then the result of operation is `false`
+
+```
+  set-column validjson json:IsValid(malformed_json)
+```
+
+## IsNull
+Returns `true` if json is `json-null` object, else `false`
+
+### Namespace
+`json`
+
+### Input
+`json-element`
+
+### Output
+boolean(`true`|`false`)
+
+### Example
+
+if `json` has the input json specified, then the result of the operation would return `false`
+
+```
+  set-column parsedjson json:Parse(json)
+  set-column notnull json:IsNull(parsedjson)
+```
+
+When a malformed json `malformed_json` is parsed, then the result of operation is `true`
+
+```
+  set-column parsedjson json:Parse(json) // Would return null as json is invalid.
+  set-column null json:IsNull(parsedjson)
+```
+
+## IsObject
+Returns `true` if json is `json-object` object, else `false`
+
+### Namespace
+`json`
+
+### Input
+`json-element`
+
+### Output
+boolean(`true`|`false`)
+
+### Example
+
+if `json` has the input json specified, then the result of the operation would return `true`.
```

Comment:
should be consistent with the examples. Some have 'json' as the field, others have 'body'. 'body' seems slightly better, as the term 'json' is mentioned a bunch of times.

Reply:
Fixed